packages/composerize/src/logic.js

```diff
@@ -38,7 +38,7 @@ export const getComposeEntry = (mapping: Mapping, value: RawValue): ComposeEntry
     if (mapping.type === 'Switch') {
         return ({
             path: mapping.path,
-            value: value === 'true' || value === true,
+            value: value === true,
```

Comment:
hmm can't remember why this exists - I think maybe cos we want to interpret a stringy boolean as an actual boolean. something something https://github.com/magicmark/composerize/issues/12

 Are you suggesting this codepath will never now be hit?
 
 until we understand this change (it's covered by tests or we get missing coverage) I'd like to keep it as is
```suggestion
            value: value === 'true' || value === true,
```